src/routines/member.ts

```diff
@@ -1,4 +1,4 @@
 import createRoutine from './utils';
 
-// eslint-disable-next-line import/prefer-default-export
 export const signOutRoutine = createRoutine('SIGN_OUT');
+export const editMemberRoutine = createRoutine('EDIT_MEMBER');
```

Comment:
Just want to make sure we get the nomenclature right. So `PATCH` and `PUT` will always be `EDIT`, correct? Let's make sure we don't mix it up with `UPDATE` or something similar.